### PR TITLE
refactor(domain.general.informations): use oui-action-menu & oui-status

### PR DESCRIPTION
**Pull request by TedKac**
## refactor(domain.general.informations): use oui-action-menu & oui-status

MFRWW-1019

### Description of the Change

Improve domain dashboard with a first iteration by using oui-action-menu & oui-status to follow the guideline.

### Benefits

improve coherence of the control panel
use ui-kit : `oui-action-menu` & `oui-status`

### Applicable Issues

* [x] need to fix tracking on `oui-action-menu-item`

### CODE REVIEW

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 1]
```diff
@@ -5,24 +5,24 @@
...
+                        <h3 class="oui-header_4 oui-tile__title" data-i18n-static="domain_dashboard_general_informations"></h3>
                         <div class="oui-tile__body">
                             <ul class="list-unstyled">
                                 <!-- OFFER -->
                                 <li class="oui-tile__item" data-ng-if="!ctrlTabGeneralInformations.isAllDom">
-                                    <div class="oui-tile__definition">
-                                        <strong data-i18n-static="domain_dashboard_offer"></strong>
-                                        <div class="text-capitalize"
+                                    <div class="oui-tile__term">
+                                        <div data-i18n-static="domain_dashboard_offer"></div>
```
Comment (FredericEspiau) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: Add class `font-weight-bold` if needed :)

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 2]
```diff
@@ -5,24 +5,24 @@
...
                                 </li>
                                 <!-- OFFER ALLDOM -->
                                 <li class="oui-tile__item"
                                     data-ng-if="ctrlTabGeneralInformations.isAllDom && ctrlTabGeneralInformations.allDom">
-                                    <div class="oui-tile__definition">
-                                        <strong
-                                            data-ng-bind="::tr('domain_dashboard_type_alldom_' + ctrlTabGeneralInformations.constructor.parseType(ctrlTabGeneralInformations.allDom.type))"></strong>
-                                        <ul class="list-unstyled">
+                                    <div class="oui-tile__term">
+                                        <div
```
Comment (FredericEspiau) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: Add class `font-weight-bold` if needed :)

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 3]
```diff
@@ -66,13 +63,24 @@ <h3 class="oui-tile__title" data-i18n-static="domain_dashboard_general_informati
                                             </div>
                                         </div>
                                     </div>
+                                    <oui-action-menu data-compact data-align="end">
+                                            <oui-action-menu-item data-ng-if="ctrlTabGeneralInformations.dnsStatus.isOk && ctrlTabGeneralInformations.dnsStatus.isHosted"
+                                                data-text="{{:: tr('domain_dashboard_dns_management_dns_details') }}"
+                                                data-on-click="ctrlTabGeneralInformations.goToDnsManagement()"></oui-action-menu-item>
+                                            <oui-action-menu-item data-ng-if="ctrlTabGeneralInformations.dnsStatus.isOk && !ctrlTabGeneralInformations.dnsStatus.isHosted"
```
Comment (FredericEspiau) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: This item and the item below could be merged maybe

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 4]
```diff
@@ -91,24 +99,20 @@ <h3 class="oui-tile__title" data-i18n-static="domain_dashboard_general_informati
...
-                                                    data-ng-click="setAction('anycast/domain-dns-anycast')"
-                                                    data-ng-if="ctrlTabGeneralInformations.domain.hasDnsAnycast === false"
-                                                    data-track-on="click"
-                                                    data-track-name="web::domain::general-informations::activation-dns-anycast"
-                                                    data-track-click="action">
-                                            </button>
                                         </div>
                                     </div>
+                                    <oui-action-menu data-compact data-align="end"
+                                                     data-ng-if="ctrlTabGeneralInformations.domain.hasDnsAnycast === false">
```
Comment (FredericEspiau) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: `ctrlTabGeneralInformations.domain.hasDnsAnycast === false` => `!ctrlTabGeneralInformations.domain.hasDnsAnycast`

Comment (FredericEspiau) [reply client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: If possible

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html (hunk 1 above)
Comment (TedKac) [reply client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: regarding the new guideline, text on bold is not necessary :)

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html (hunk 2 above)
Comment (TedKac) [reply client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: regarding the new guideline, text on bold is not necessary :)

Comment (FredericEspiau) [reply client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: "new"

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 5]
```diff
@@ -64,14 +64,7 @@ <h3 class="oui-header_4 oui-tile__title" data-i18n-static="domain_dashboard_gene
...
-                                                data-text="{{:: tr('domain_dashboard_dns_management_dns_details') }}"
-                                                data-on-click="ctrlTabGeneralInformations.goToDnsManagement()"></oui-action-menu-item>
-                                            <oui-action-menu-item data-ng-if="ctrlTabGeneralInformations.dnsStatus.isOk && !ctrlTabGeneralInformations.dnsStatus.isHosted"
-                                                data-text="{{:: tr('domain_dashboard_dns_management_dns_details') }}"
-                                                data-on-click="ctrlTabGeneralInformations.goToDnsManagement()"></oui-action-menu-item>
-                                            <oui-action-menu-item data-ng-if="!ctrlTabGeneralInformations.dnsStatus.isOk"
-                                                data-text="{{:: tr('domain_dashboard_dns_management_dns_details') }}"
+                                            <oui-action-menu-item data-text="{{:: tr('domain_dashboard_dns_management_dns_details') }}"
```
Comment (FredericEspiau) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: 😄

**Pull request merged**